src/fsharp/ast.fs

```diff
@@ -1742,7 +1755,7 @@ let mkSynDotBrackSeqSliceGet  m mDot arr (argslist:list<SynIndexerArg>) =
                        | SynIndexerArg.One x -> yield x
                        | _ -> () ]
     if notsliced.Length = argslist.Length then
-        SynExpr.DotIndexedGet(arr,[SynIndexerArg.One (SynExpr.Tuple(notsliced,[],unionRanges (Seq.head notsliced).Range (Seq.last notsliced).Range))],mDot,m)
```

Comment:
Change the code in the compiler to use List.head and List.last since we define List.last in src\absil\illib.fs in all situations. (If you do add something like Seq.last to the compiler, add it in src\absil\illib.fs)